ncm-opennebula/src/main/perl/opennebula.pm

```diff
@@ -325,6 +330,19 @@ sub change_opennebula_passwd
     return 1;
 }
 
+# Sync hyps VMMs scripts
+sub sync_opennebula_hyps
+{
+    my ($self, $cmd) = @_;
+    my $output;
+
+    $output = $self->run_onehost_as_oneadmin_with_ssh($cmd, "localhost", 0);
+    if (!$output) {
+        $self->error("Quattor unable to execute onehost command.");
```

Comment:
error message should mention what was trying to do (likethe info message does), not only what failed.